backend/config/.env.example

```
@@ -159,3 +161,14 @@ FITBIT_DEFAULT_SCOPE=activity heartrate sleep profile
 ULTRAHUMAN_CLIENT_ID=your-ultrahuman-client-id
 ULTRAHUMAN_CLIENT_SECRET=your-ultrahuman-client-secret
 ULTRAHUMAN_DEFAULT_SCOPE=ring_data cgm_data profile
+
+#--- Google ---#
+GOOGLE_CLIENT_ID=your-google-client-id
+GOOGLE_CLIENT_SECRET=your-google-client-secret
+GOOGLE_DEFAULT_SCOPE=openid email https://www.googleapis.com/auth/googlehealth.activity_and_fitness.readonly https://www.googleapis.com/auth/googlehealth.health_metrics_and_measurements.readonly https://www.googleapis.com/auth/googlehealth.nutrition.readonly https://www.googleapis.com/auth/googlehealth.sleep.readonly https://www.googleapis.com/auth/googlehealth.settings.readonly
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)